### PR TITLE
numeral rather than pretty bytes MONGOSH-1124

### PR DESCRIPTION
**Pull request by lerouxb**
This treats 1KB as 1024 bytes, has no space between the number and unit and uses KB, MB, etc all just like legacy shell. BUT I don't know about all the nuances of 0.1GB vs 100MB, for example. How much do we care about matching that level of detail? Also unclear about things like rounding up and down.

### CODE REVIEW

File: packages/browser-repl/src/components/types/show-dbs-output.tsx [hunk 1]
```diff
@@ -12,14 +12,19 @@ type DatabaseObject = {
   map: any;
 };
 
+function formatBytes(value: number): string {
+  const precision = value <= 1000 ? '0' : '0.0';
+  return numeral(value).format(precision + 'b');
```
Comment (addaleax) [inline packages/browser-repl/src/components/types/show-dbs-output.tsx:17]:
```suggestion
  const precision = value <= 1000 ? '0' : '0.00';
  return numeral(value).format(precision + ' ib');
```

would make this closer to what we want/had previously and actually use 1024-based units, I think

File: packages/cli-repl/src/format-output.spec.ts [hunk 2]
```diff
@@ -172,7 +178,7 @@ for (const colors of [ false, true ]) {
           type: 'ShowCollectionsResult'
         }));
 
-        expect(output).to.contain('nested_documents\ndecimal128\ncoll\npeople_imported   [view]\ncats              [time-series]');
+        expect(output).to.equal('nested_documents\ndecimal128\ncoll\npeople_imported   [view]\ncats              [time-series]');
```
Comment (addaleax) [inline packages/cli-repl/src/format-output.spec.ts:181]: :+1:

**Review by addaleax — APPROVED** [review]

File: packages/cli-repl/src/format-output.spec.ts (hunk 2 above)
Comment (lerouxb) [reply packages/cli-repl/src/format-output.spec.ts:181]: accident. meant to change the ones I was changing. But if it works I suppose there's no harm ;)

**Comment by addaleax** [comment]
nb, I force-pushed because we also use conventional-commits in this repo

**Pull request merged**